docs/source/ingest/configs/embedding_config.rst

```diff
@@ -10,5 +10,6 @@ the dataset.
 
 Configs
 ---------------------
+* ``embedding_provider``: An unstructured embedding provider to use while doing embedding. A few examples: langchain-openai, langchain-huggingface, langchain-aws-bedrock.
 * ``api_key``: If an api key is required to generate the embeddings via an api (i.e. OpenAI)
```

Comment:
Will this conflict with the pinecone `api_key` ?

Reply:
Whoops. This is in source...

Reply:
I think its supposed to be embedding_api_key?

Reply:
The documentation here should be updated. All embedding fields now have an embedding prepend:
https://github.com/Unstructured-IO/unstructured/blob/30cbc420a0809d4fcde2397d9b096b1e8e638203/unstructured/ingest/cli/interfaces.py#L408-L426

Therefore, addressed with [update embedding docs to have embedding prepend for cli args](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/ca0785e97860cba9191a5b68a5a8cdff4a491146)

(answer to the actual question: even in that case, it wouldn't - since pinecone api_key is a destination connector option while embedding api_key is a source connector option)